aws_adfs/login.py

```diff
@@ -151,7 +155,7 @@ def _authenticate(config, username=None, password=None):
     del password
 
     # Decode the response
-    html = ET.fromstring(response.text.decode('utf8'), ET.HTMLParser())
+    html = ET.fromstring(response.text, ET.HTMLParser())
```

Comment:
No, you right. At least for os with UTF-8 set as a base encoding. I think, it can be safely removed.